tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts

```diff
@@ -320,7 +320,13 @@ export class TestCodeModeler {
         this.initiateTests();
         this.codeModel.operationGroups.forEach((operationGroup) => {
             operationGroup.operations.forEach((operation) => {
-                const exampleGroup = new ExampleGroup(operationGroup, operation, operationGroup.language.default.name + '_' + operation.language.default.name);
+                const operationId = operationGroup.language.default.name + '_' + operation.language.default.name;
+                const mediaTypes = operation.requests[0]?.protocol?.http?.mediaTypes;
+                if (mediaTypes && mediaTypes.indexOf('application/json') < 0) {
```

Comment:
Could you please add a comment here to elaborate why we need to do this for now?